spring-cloud-gcp-data-spanner/src/main/java/com/google/cloud/spring/data/spanner/core/SpannerTemplate.java

```diff
@@ -158,6 +159,21 @@ public long executePartitionedDmlStatement(Statement statement) {
     return rowsAffected;
   }
 
+  @Override
+  public long executePartitionedDmlStatement(Statement statement, UpdateOption... options) {
```

Comment:
Can you change `executePartitionedDmlStatement(Statement statement)` to delegated to this method with empty array of `UpdateOption`?

Reply:
sure, updated it